x-pack/plugins/maps/public/classes/sources/es_search_source/es_search_source.tsx

```diff
@@ -431,6 +441,27 @@ export class ESSearchSource extends AbstractESSource implements ITiledSingleLaye
     return matchingIndexes.length === 1;
   }
 
+  async getDefaultFields(): Promise<Record<string, string>> {
+    if (!(await this._isDrawingIndex())) {
+      return {};
+    }
+    const user = await getSecurityService()?.authc.getCurrentUser();
+    const timestamp = new Date().toISOString();
+    return {
+      ...(user ? { user: user.username } : {}),
```

Comment:
This should be discussed and agreed upon. We can really capture the user's username, full name and/or email. I opted to capture the user's username but can definitely revise if there are any strong opinions! cc @kmartastic